### PR TITLE
[eos_l3_interfaces] Fixes sort issue in presence of secondary ipv4 address

### LINKED ISSUE #67: eos_l3_interfaces : override operation in presence of a secondary ipv4 address errors out
##### SUMMARY
When the device interface has a secondary ipv4 address, and an override operation is called with the same config, the play erros out.

Issue seen while implementing round trip test.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
eos_l3_interfaces

##### ANSIBLE VERSION

```
ansible 2.11.0.dev0
  config file = /etc/ansible/ansible.cfg


```

##### CONFIGURATION

```
HOST_KEY_CHECKING(env: ANSIBLE_HOST_KEY_CHECKING) = False
INTERPRETER_PYTHON(env: ANSIBLE_PYTHON_INTERPRETER) = /usr/bin/env python
PERSISTENT_COMMAND_TIMEOUT(env: ANSIBLE_PERSISTENT_COMMAND_TIMEOUT) = 500
PERSISTENT_CONNECT_TIMEOUT(env: ANSIBLE_PERSISTENT_CONNECT_TIMEOUT) = 500


```

##### STEPS TO REPRODUCE

Config on the device:
interface Ethernet1
   ip address 192.0.2.12/24
   ip address 203.0.113.27/31 secondary
   no ipv6 address
interface Ethernet2
   no ip address
   ipv6 address 2001:db8::1/64
interface Management1
   ip address dhcp
   no ipv6 address

Playbook:
- set_fact:
    config1:

      - name: Ethernet1
        ipv4:

          - address: 198.51.100.14/24

    config2:

      - name: Ethernet2
        ipv4:

          - address: 203.0.113.227/31
- name: Merge provided configuration with device configuration (base config).
  register: baseconfig
  become: true
  arista.eos.eos_l3_interfaces:
    config: '{{ config1 }}'
    state: merged

- become: true
  arista.eos.eos_facts:
    gather_network_resources: l3_interfaces

- name: Merge provided configuration with device configuration (config to be reverted).
  register: result
  become: true
  arista.eos.eos_l3_interfaces:
    config: '{{ config2 }}'
    state: merged
- name: Revert back to base config using facts round trip
  become: true
  register: revert
  arista.eos.eos_l3_interfaces:
    config: "{{ ansible_facts['network_resources']['l3_interfaces'] }}"
    state: overridden

##### EXPECTED RESULTS

The play should return success
##### ACTUAL RESULTS

```
The full traceback is:
Traceback (most recent call last):
  File "/home/gosriniv/.ansible/tmp/ansible-local-10264q7vwaquw/ansible-tmp-1593550050.2196186-10608-102607079081219/AnsiballZ_eos_l3_interfaces.py", line 116, in <module>
    _ansiballz_main()
  File "/home/gosriniv/.ansible/tmp/ansible-local-10264q7vwaquw/ansible-tmp-1593550050.2196186-10608-102607079081219/AnsiballZ_eos_l3_interfaces.py", line 108, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/home/gosriniv/.ansible/tmp/ansible-local-10264q7vwaquw/ansible-tmp-1593550050.2196186-10608-102607079081219/AnsiballZ_eos_l3_interfaces.py", line 54, in invoke_module
    runpy.run_module(mod_name='ansible_collections.arista.eos.plugins.modules.eos_l3_interfaces', init_globals=None, run_name='__main__', alter_sys=True)
  File "/usr/lib64/python2.7/runpy.py", line 188, in run_module
    fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/tmp/ansible_arista.eos.eos_l3_interfaces_payload_etu4sD/ansible_arista.eos.eos_l3_interfaces_payload.zip/ansible_collections/arista/eos/plugins/modules/eos_l3_interfaces.py", line 405, in <module>
  File "/tmp/ansible_arista.eos.eos_l3_interfaces_payload_etu4sD/ansible_arista.eos.eos_l3_interfaces_payload.zip/ansible_collections/arista/eos/plugins/modules/eos_l3_interfaces.py", line 400, in main
  File "/tmp/ansible_arista.eos.eos_l3_interfaces_payload_etu4sD/ansible_arista.eos.eos_l3_interfaces_payload.zip/ansible_collections/arista/eos/plugins/module_utils/network/eos/config/l3_interfaces/l3_interfaces.py", line 77, in execute_module
  File "/tmp/ansible_arista.eos.eos_l3_interfaces_payload_etu4sD/ansible_arista.eos.eos_l3_interfaces_payload.zip/ansible/module_utils/connection.py", line 195, in __rpc__
ansible.module_utils.connection.ConnectionError:  203.0.113.27/31 secondary
% Invalid input
veos(config-s-ansibl-if-Et1)#
fatal: [192.168.122.113]: FAILED! => {
    "changed": false,
    "module_stderr": "Traceback (most recent call last):\n  File \"/home/gosriniv/.ansible/tmp/ansible-local-10264q7vwaquw/ansible-tmp-1593550050.2196186-10608-102607079081219/AnsiballZ_eos_l3_interfaces.py\", line 116, in <module>\n    _ansiballz_main()\n  File \"/home/gosriniv/.ansible/tmp/ansible-local-10264q7vwaquw/ansible-tmp-1593550050.2196186-10608-102607079081219/AnsiballZ_eos_l3_interfaces.py\", line 108, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/gosriniv/.ansible/tmp/ansible-local-10264q7vwaquw/ansible-tmp-1593550050.2196186-10608-102607079081219/AnsiballZ_eos_l3_interfaces.py\", line 54, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.arista.eos.plugins.modules.eos_l3_interfaces', init_globals=None, run_name='__main__', alter_sys=True)\n  File \"/usr/lib64/python2.7/runpy.py\", line 188, in run_module\n    fname, loader, pkg_name)\n  File \"/usr/lib64/python2.7/runpy.py\", line 82, in _run_module_code\n    mod_name, mod_fname, mod_loader, pkg_name)\n  File \"/usr/lib64/python2.7/runpy.py\", line 72, in _run_code\n    exec code in run_globals\n  File \"/tmp/ansible_arista.eos.eos_l3_interfaces_payload_etu4sD/ansible_arista.eos.eos_l3_interfaces_payload.zip/ansible_collections/arista/eos/plugins/modules/eos_l3_interfaces.py\", line 405, in <module>\n  File \"/tmp/ansible_arista.eos.eos_l3_interfaces_payload_etu4sD/ansible_arista.eos.eos_l3_interfaces_payload.zip/ansible_collections/arista/eos/plugins/modules/eos_l3_interfaces.py\", line 400, in main\n  File \"/tmp/ansible_arista.eos.eos_l3_interfaces_payload_etu4sD/ansible_arista.eos.eos_l3_interfaces_payload.zip/ansible_collections/arista/eos/plugins/module_utils/network/eos/config/l3_interfaces/l3_interfaces.py\", line 77, in execute_module\n  File \"/tmp/ansible_arista.eos.eos_l3_interfaces_payload_etu4sD/ansible_arista.eos.eos_l3_interfaces_payload.zip/ansible/module_utils/connection.py\", line 195, in __rpc__\nansible.module_utils.connection.ConnectionError:  203.0.113.27/31 secondary\r\n% Invalid input\r\nveos(config-s-ansibl-if-Et1)#\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",


```

### PR DESCRIPTION
**Pull request by GomathiselviS**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/611
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/612
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY
Fixes #67 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
eos_l3_interfaces.py
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: tests/integration/targets/eos_l3_interfaces/tests/common/rtt.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  become: true
+  register: revert
+  arista.eos.eos_l3_interfaces:
+    config: "{{ ansible_facts['network_resources']['l3_interfaces'] }}"
+    state: overridden
+
+- name: Assert that config was reverted
+  assert:
+    that: "{{ baseconfig['after'] | symmetric_difference(revert['after']) |length\
+      \ == 0 }}"
```
Comment (Qalthos) [inline tests/integration/targets/eos_l3_interfaces/tests/common/rtt.yaml:83]:
```suggestion
    that: "{{ baseconfig['after'] | symmetric_difference(revert['after']) == [] }}"
```

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Pull request merged**